### PR TITLE
Os ubuntu 20 04

### PR DESCRIPTION
**Pull request by marcelaconfig**

### CODE REVIEW

File: catsgo.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import time
 import sys
 import logging
+import traceback
```
Comment (JezSw) [inline catsgo.py:9]: Is this still used?

File: catsgo.py [hunk 2]
```diff
@@ -32,6 +33,7 @@ def login():
     try:
         load_cookies()
     except:
+        logging.error(f'''Login {traceback.format_exc()}''')
```
Comment (JezSw) [inline catsgo.py:36]: We would usually use single (') or double quoted (") format strings. Not sure what's best, but we should be consistent when triple quoted isn't NEEDED.

**Review by JezSw — COMMENTED** [review]
Nothing major but some comments within that might want addressing before merge

File: catsgo.py (hunk 1 above)
Comment (marcelaconfig) [reply catsgo.py:9]: Yes, I need the traceback imported so I can have good description of exceptions

File: catsgo.py (hunk 2 above)
Comment (JezSw) [reply catsgo.py:36]: This probably should be done too

**Review by JezSw — APPROVED** [review]

**Pull request merged**